app/src/main/java/org/sopt/dosopttemplate/HomeFragment.kt

```diff
@@ -0,0 +1,86 @@
+package org.sopt.dosopttemplate
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import androidx.recyclerview.widget.ConcatAdapter
+import org.sopt.dosopttemplate.databinding.FragmentHomeBinding
+
+class HomeFragment: Fragment() {
+    private var _binding : FragmentHomeBinding? = null
+    private val binding: FragmentHomeBinding
+        get() = requireNotNull(_binding){ "바인딩에러" }
+
+
+    private val mockFriendList =listOf<Friend>(
+        Friend(
+            profileImage = R.drawable.img,
+            name = "라이언",
+            self_description = "",
+        ),
+        Friend(
+            profileImage = R.drawable.img_1,
+            name = "양파쿵야",
+            self_description = "돈벌기 힘들다",
+        ),
+        Friend(
+            profileImage = R.drawable.img_2,
+            name = "스누피친구",
+            self_description = "내 이름이 머더라",
+        ),
+        Friend(
+            profileImage = R.drawable.img_3,
+            name = "신짱구",
+            self_description = "어 이쁜 누나다",
+        ),
+        Friend(
+            profileImage = R.drawable.img_4,
+            name = "김철수",
+            self_description = "",
+        ),
+        Friend(
+            profileImage = R.drawable.img_5,
+            name = "흰둥이",
+            self_description = "",
+        ),
+        Friend(
+            profileImage = R.drawable.img_6,
+            name = "액션가면",
+            self_description = "하하ㅏ",
+        ),
+        Friend(
+            profileImage = R.drawable.img_7,
+            name = "부리부리대마왕",
+            self_description = "",
+        )
+    )
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
```

Comment:
```suggestion
    ): View {
```
반환하는 View가 null일 수 없기 때문에 ?를 지워도 됩니둥